.github/workflows/test.yml

```
@@ -1,5 +1,12 @@
-name: Deploy To Development
-on: [workflow_dispatch]
+name: Test
+run-name: Test - ${{ github.head_ref || github.ref_name }} to ${{ inputs.environment }} by @${{ github.actor }}
```

actionlint (1.7.7)

property "environment" is not defined in object type {}

(expression)